src/testIntegration/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/cjsm/CjsmResourceTest.kt

```diff
@@ -30,6 +30,6 @@ class CjsmResourceTest : IntegrationTest() {
 
     val cjsmDirectorySize = cjsmDirectoryRepository.count()
 
-    assertThat(cjsmDirectorySize).isEqualTo(1000)
+    assertThat(cjsmDirectorySize).isEqualTo(100)
```

Comment:
Make the test run quicker - otherwise need to start playing with WebTestClient's timeout which is hard than it needs to be :(